scripts/main.sh

```diff
@@ -141,6 +141,11 @@ STATUS() {
 PRE_CHECK $1
 case $1 in
     init)
+        WD=$PWD
+        BRIDGE=./bridge
+        cd $BRIDGE &&
+        npm install &&
```

Comment:
I believe it's better to use `npm ci` instead to make sure that we are getting the exact versions in the package-lock.json file.

Reply:
done